rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -2031,7 +2031,7 @@ def create_container_using_swift(container_name, rgw, user_info):
         )
 
 
-def test_bucket_stats_across_sites(bucket_name_to_create):
+def test_bucket_stats_across_sites(bucket_name_to_create, config):
```

Comment:
@viduship, config option added might cause regression in suites where the function is called. Can we set a default value of None/False to avoid it.

Reply:
thankyou for the review, I have updated the function where it is called.